src/acl/agent.ts

```diff
@@ -37,59 +37,59 @@ import {
   internal_getAccessModes,
   internal_combineAccessModes,
   unstable_hasResourceAcl,
-  unstable_getResourceAcl,
   unstable_hasFallbackAcl,
-  unstable_getFallbackAcl,
 } from "../acl";
 
 export type unstable_AgentAccess = Record<WebId, unstable_AccessModes>;
 
 /**
- * Find out what Access Modes have been granted to a given Agent specifically for a given LitDataset.
+ * Find out what Access Modes have been granted to a given Agent specifically for a given Resource.
  *
  * Keep in mind that this function will not tell you what access the given Agent has through other ACL rules, e.g. public or group-specific permissions.
  *
  * Also, please note that this function is still experimental: its API can change in non-major releases.
  *
- * @param dataset The LitDataset to which the given Agent may have been granted access.
+ * @param resourceInfo Informations about the Resource to which Agents may have been granted access.
```

Comment:
```suggestion
 * @param resourceInfo Information about the Resource to which the given Agent may have been granted access.
```

(Why did you change this to "Agent_s_" - this function operates on a single Agent, right?

Reply:
That's an oversight, I'm not sure how I ended up rephrasing this, thanks for catching it.